js/video/controllers/video.controller.js

```diff
@@ -176,6 +177,11 @@
             loadingTimeout = $timeout(function () {
                 vm.loading = false;
             }, 2000);
+            
+            feedIsPresentInPlaylist = dataStore.feeds.map(
```

Comment:
`feedIsPresentInPlaylist` is a bit confusing. Feed is the response of the API which contains a playlist.